api/v1beta1/nodepool_types.go

```diff
@@ -564,6 +566,15 @@ type KubevirtCompute struct {
 	// +optional
 	// +kubebuilder:default=2
 	Cores *uint32 `json:"cores"`
+
+	// QosClass If set to "Guaranteed", requests the scheduler to place the VirtualMachineInstance on a node with
+	// limit memory and CPU, equal to be the requested values, to set the VMI as a Guaranteed QoS Class;
+	// See here for more details:
+	// https://kubevirt.io/user-guide/operations/node_overcommit/#requesting-the-right-qos-class-for-virtualmachineinstances
+	//
+	// +optional
+	//+kubebuilder:validation:Enum=Guaranteed
+	QosClass *string `json:"qosClass,omitempty"`
```

Comment:
i think we should use a type here. and set the default to Burstable so the field is better understood.